trieste/utils/pareto.py

```diff
@@ -105,3 +105,32 @@ def _bounds_2d(front: TensorType) -> BoundedVolumes:
         upper = tf.concat([range_ + 1, pf_ext_idx[::-1, 1:][: pf_ext_idx[-1, 0]]], axis=-1)
 
         return BoundedVolumes(lower, upper)
+
+    def hypervolume_indicator(self, reference: tf.Tensor) -> tf.Tensor:
+        """
+        Method to calculate the hypervolume indicator
+        The hypervolume indicator is the volume of the dominated region.
+
+        :param reference: a reference point to use, with shape [D].
+            Should be equal or bigger than the anti-ideal point of the Pareto set
+            for comparing results across runs, the same reference point must be used.
```

Comment:
Is this meant to be
```
            Should be equal or bigger than the anti-ideal point of the Pareto set.
            For comparing results across runs, the same reference point must be used.
```
or
```
            Should be equal or bigger than the anti-ideal point of the Pareto set
            for comparing results across runs. The same reference point must be used.
```

Reply:
The first one, I've updated the docstring.